### PR TITLE
Add check for empty messages in delete_message_from_queue

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: cloudigrade/util/aws/sqs.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ def delete_message_from_queue(queue_url, messages):
         dict: The response from the delete call.
 
     """
+    if not bool(messages):
```
Comment (infinitewarp) [inline cloudigrade/util/aws/sqs.py]: Why are you explicitly casting this to a `bool`? `if not messages` should work for an empty list.

**Comment by elyezer** [comment]
> Should we add a test?

:heart:

File: cloudigrade/util/aws/sqs.py (hunk 1 above)
Comment (elyezer) [reply cloudigrade/util/aws/sqs.py]: I got curious about this too. But asked on Slack. Better to talk here instead.

Comment (ilyavw) [inline cloudigrade/util/aws/sqs.py]: In addition to adding a test for this, also adding an info/debug log would be helpful.

**Comment by ilyavw** [comment]
https://github.com/cloudigrade/cloudigrade/pull/407#issuecomment-403512811
Shouldn't it be QE asking this question? 😉

File: cloudigrade/util/aws/sqs.py (hunk 1 above)
Comment (kholdaway) [reply cloudigrade/util/aws/sqs.py]: I can remove that.  Great catch.

Comment (kholdaway) [reply cloudigrade/util/aws/sqs.py]: This happens every time the analyzer runs if instances haven't changed.  I can add this but it will be common.

Comment (ilyavw) [reply cloudigrade/util/aws/sqs.py]: Sounds like a debug log would be appropriate then.

**Comment by kholdaway** [comment]
@abaiken @infinitewarp @ilyavw Back at you

File: cloudigrade/util/aws/sqs.py [hunk 2]
```diff
@@ -39,6 +43,11 @@ def delete_message_from_queue(queue_url, messages):
         dict: The response from the delete call.
 
     """
+    if not messages:
+        logger.debug(
+            _('No messages to delete because messages list was empty.'))
```
[the remark this answers is not in the document]
Comment (kholdaway) [reply cloudigrade/util/aws/sqs.py]: @infinitewarp done.

**Comment by elyezer** [comment]
https://github.com/cloudigrade/cloudigrade/pull/407#issuecomment-403512811
Shouldn't it be QE asking this question? 😉

We were checking if Devs would ask for it :1st_place_medal:

**Pull request merged**